gatsby/static/css/webflow-overrides.css

```diff
@@ -255,6 +278,10 @@ label {
   width: 100%;
 }
 
+.mxcontent__main {
+  line-height: 1.8em;
+}
+
```

Comment:
Maybe this should go in [`gatsby/static/css/matrix-org.webflow.css#L283`](https://github.com/matrix-org/matrix.org/blob/1397f009d4474a7001071bec5648a05cb58e0684/gatsby/static/css/matrix-org.webflow.css#L283)